python/ray/serve/scripts.py

```diff
@@ -228,6 +266,8 @@ def run(config_or_import_path: str, address: str):
 
             schematized_config = ServeApplicationSchema.parse_obj(config)
             deployments = schema_to_serve_application(schematized_config)
+
+            serve.start()
```

Comment:
is this necessary here? it's not in here before

Reply:
See [above reply](https://github.com/ray-project/ray/pull/22805#discussion_r819025844).

Reply:
can we at least pull this out out of if else? It seems like we are calling it in both cases